src/utils/index.ts

```diff
@@ -53,3 +54,19 @@ export const addClassNameIfExist = (className?: string): string => {
 
   return className;
 };
+
+export const getCmsContentWithSlug = (context: __WebpackModuleApi.RequireContext, type?: NewsTag): any[] => {
```

Comment:
forgotten about `NewsTag`arg  was for what, maybe leave a comment it's for the tag filter functionality in news page?

Reply:
to categorise the news

Reply:
ok will add a comment for that